test/external/run-all-tests

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# Run all the external tests
+
+# Fail if any command fails
+set -e 
+set -o pipefail
+
+# Tests all the variations (JS-TS and CJS-ESM) with the `express-rate-limit` package.
+cd imports/
+for dir in `ls -d */`; do
+	cd $dir
+
+	rm -rf node_modules
+	npm install ../../../../*.tgz
```

Comment:
What is this?

I assume it's the node_modules folder.. but why is it in a .tgz file? Maybe add a comment clarifying that?

Reply:
Basically all of the tests in the `test/external` folder are example 'projects', that have `express-rate-limit` as a dependency. When we run `npm pack`, it builds a `.tgz` file that we install as a dependency in all the example projects, using `npm install <path-to-tgz>`. The file name contains the version number, so to allow the script to work no matter what the version, i use a `*.tgz` glob. I guess I could make it more specific and change it to `express-rate-limit-*.tgz`, and add this as a comment in that script.

Reply:
Done in b548dc8